code/modules/mob/living/basic/icemoon/ice_whelp/ice_whelp_abilities.dm

```diff
@@ -0,0 +1,33 @@
+/datum/action/cooldown/ice_breathe
```

Comment:
I believe this should subtype `/mob_cooldown`

Reply:
issue with /mob_cooldown is it renders the mob to use its melee attacks until the cooldown expires. ive also noticed basic goliaths use this same type